packages/scripts/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 3.1.0 (unreleased)
```

Comment:
This should have been 4.0.0 as it's a breaking change I think.

Reply:
I don't think it is, previously we would exit when webpack config wasn't provided with the following message:
```js
console.log( 'Webpack config file is missing.' );
```

Reply:
Also the exit code has changed. I see it as non-breaking because that was an unsupported use case that we now support (not the other way around). But I see how could be argued that this is also major.